persistence/types/account_shared_sql.go

```diff
@@ -0,0 +1,36 @@
+package types
+
```

Comment:
```suggestion
// CLEANUP: Move SQL specific business logic into a `sql` package under `persistence`
```

@h5law Also, feel free to resolve comments (i.e. suggestions like these) after you apply them (either through github or yourself). There's always some inherent trust between the reviewer and reviewee.